google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -347,7 +359,21 @@ protected void textChanged(DocumentEvent event) {
   private void setManagedSdkUiAvailable(boolean available) {
```

Comment:
is this method only dealing with setting the "update" related portions of the UI? if so maybe the name should be updated to reflect this.

Reply:
if you could show a screenshot in the PR that would help me visualize it

Reply:
correct, for Managed SDK it's update, for custom SDK it's SDK path, those two methods are generally used to switch between active UI for managed/custom SDK, not for specific update or path portion though. Working on the screenshot.

Reply:
Updated description with screenshots.